backend/src/routes/api/cluster-settings/clusterSettingsUtils.ts

```diff
@@ -0,0 +1,57 @@
+import { FastifyRequest } from 'fastify';
+import { KubeFastifyInstance, ClusterSettings } from '../../../types';
+
+export const updateClusterSettings = async (
+  fastify: KubeFastifyInstance,
+  request: FastifyRequest,
+): Promise<ClusterSettings | string> => {
+  const coreV1Api = fastify.kube.coreV1Api;
+  const namespace = fastify.kube.namespace;
+  const query = request.query as { [key: string]: string };
+  try {
+    if (query.pvcSize) {
+      await coreV1Api.patchNamespacedConfigMap(
+        'jupyterhub-cfg',
+        namespace,
+        {
+          data: { singleuser_pvc_size: `${query.pvcSize}Gi` },
+        },
+        undefined,
+        undefined,
+        undefined,
+        undefined,
+        {
+          headers: {
+            'Content-Type': 'application/merge-patch+json',
+          },
+        },
+      );
+    }
+    return {
+      pvcSize: Number(query.pvcSize),
```

Comment:
When calling update API, we don't need to return the updated value back to the frontend. Instead, we could return an object like {success: boolean, error: string} to indicate whether it's successful or not, so the frontend could show the alert based on the returned value. You can refer to this function https://github.com/opendatahub-io/odh-dashboard/blob/master/backend/src/routes/api/components/list.ts#L34-L65

Reply:
Done.